xtensorConfig.cmake.in

```diff
@@ -19,15 +19,44 @@
 include(CMakeFindDependencyMacro)
 find_dependency(xtl @xtl_REQUIRED_VERSION@)
 
-if(XTENSOR_USE_XSIMD)
-    find_dependency(xsimd @xsimd_REQUIRED_VERSION@)
+if(NOT TARGET @PROJECT_NAME@)
+    include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+    get_target_property(@PROJECT_NAME@_INCLUDE_DIRS @PROJECT_NAME@ INTERFACE_INCLUDE_DIRECTORIES)
+    if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.8)
+        target_compile_features(@PROJECT_NAME@ INTERFACE cxx_std_14)
+    endif()
 endif()
 
-if(XTENSOR_USE_TBB)
+if(XTENSOR_USE_XSIMD AND XTENSOR_USE_TBB)
```

Comment:
Why is this necessary if you have two independent sections (one for `XSIMD` and one for `TBB`?